les-4-opdracht-transition/styles.css

```diff
@@ -3,12 +3,23 @@
     --menu-bar-height: 10px;
 }
 
+#container{
```

Comment:
Achter je selector #container hoort een spatie voor de curly bracket, met ctrl+alt+l of command+alt+l op Mac laat je webstorm dit voor jou doen.